plotly/graph_reference.py

```diff
@@ -68,9 +68,62 @@ def get_graph_reference():
     """
     path = os.path.join('package_data', 'default-schema.json')
     s = resource_string('plotly', path).decode('utf-8')
-    graph_reference = _json.loads(s)
+    graph_reference = utils.decode_unicode(_json.loads(s))
+
+    # TODO: Patch in frames info until it hits streambed. See #659
+    graph_reference['frames'] = {
```

Comment:
Just patching in what I'm expecting to see here. This means that we don't need to wait for any changes to land on Plotly prod and can get this into `1.X` *now*. I'll remove this once that changes.